### PR TITLE
feat(terraform): New checks

### PR DESCRIPTION
**Pull request by tsmithv11**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

[//]: # "
    # PR Title
    We use the title to create changelog automatically and therefore only allow specific prefixes
    - break:    to indicate a breaking change, this supersedes any of the other types
    - feat:     to indicate new features or checks
    - fix:      to indicate a bugfix or handling of edge cases of existing checks
    - docs:     to indicate an update to our documentation
    - chore:    to indicate adjustments to workflow files or dependency updates
    - platform: to indicate a change needed for the platform
    Each prefix should be accompanied by a scope that specifies the targeted framework. If uncertain, use 'general'.
    #    
    Allowed prefixs:
    ansible|argo|arm|azure|bicep|bitbucket|circleci|cloudformation|dockerfile|github|gha|gitlab|helm|kubernetes|kustomize|openapi|sast|sca|secrets|serverless|terraform|general|graph|terraform_plan|terraform_json
    #
    ex.
    feat(terraform): add CKV_AWS_123 to ensure that VPC Endpoint Service is configured for Manual Acceptance
"

## Description

Open a policy to match runtime policies

`CKV_AWS_375` matches `43c42760-5283-4bc4-ac43-a80e58c4139f`
`CKV_AWS_376` matches `836a7c8c-34c2-4861-be1e-df2f8cd27aab`
`CKV_AWS_377` and `CKV_AWS_378` don't have a mapped policy

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: tests/terraform/runner/test_plan_runner.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ def test_plan_runner_with_empty_vpc_connection(self):
         self.assertEqual(report.get_exit_code({'soft_fail': False, 'soft_fail_checks': [], 'soft_fail_threshold': None, 'hard_fail_checks': [], 'hard_fail_threshold': None}), 1)
         self.assertEqual(report.get_exit_code({'soft_fail': True, 'soft_fail_checks': [], 'soft_fail_threshold': None, 'hard_fail_checks': [], 'hard_fail_threshold': None}), 0)
 
-        self.assertEqual(report.get_summary()["failed"], 106)
+        self.assertEqual(report.get_summary()["failed"], 107)
```
Comment (tsmithv11) [inline tests/terraform/runner/test_plan_runner.py:279]: Fails `CKV_AWS_376` because there is a listener with `http`

**Review by ChanochShayner — APPROVED** [review]

**Review by itai1357 — APPROVED** [review]
great tests!

**Comment by Shanjohn** [comment]
hi @tsmithv11 does CKV_AWS_378 prohibit ssl offload on load balancer level for things like ECS Services? Is this intended?

**Comment by tsmithv11** [comment]
@Shanjohn can you open an Issue and tag me with an example that reproduces the false positive and I'll improve this check?

**Pull request merged**